waku/v2/protocol/waku_noise/noise.nim

```diff
@@ -98,10 +261,276 @@ proc genKeyPair*(rng: var BrHmacDrbgContext): KeyPair =
   keyPair.publicKey = keyPair.privateKey.public()
   return keyPair
 
+# Prints Handshake Patterns using Noise pattern layout
+proc print*(self: HandshakePattern) 
+   {.raises: [IOError, NoiseMalformedHandshake].}=
+  try:
+    if self.name != "":
+      echo self.name, ":"
```

Comment:
An `echo` is left behind!

Reply:
It was a quick way to have a new line at the end! I made it consistent with a `stdout.write`